src/components/TrendingCompanies/CompanyCard.vue

```diff
@@ -26,21 +38,42 @@
 export default {
   name: 'CompanyCard',
   props: {
-    title: {
+    company: {
       type: String,
       default: ''
     },
-    subtitle: {
+    ticker: {
       type: String,
       default: ''
     },
     imgsrc: {
       type: String,
       default: ''
     },
-    sold: {
-      type: Boolean,
-      default: false
+    price: {
+      type: Number,
+      default: 0
+    },
+    variation: {
+      type: Number,
+      default: 0
+    }
+  },
+  data () {
+    return {
+      stockcolor: ''
+    }
+  },
+  beforeMount () {
+    this.stockColorPicker()
+  },
+  methods: {
+    stockColorPicker () {
+      if (this.variation > 0) {
+        this.stockcolor = 'green'
+      } else {
+        this.stockcolor = 'red'
+      }
```

Comment:
just a trick 😎 
```javascript 
this.variation > 0 ? this.stockcolor = 'green' : this.stockcolor = 'red'
```

Reply:
ok thank you

Reply:
doing it this way is giving me a major code smell